README.md

```diff
@@ -1,2 +1,116 @@
+Therefore, having a visualization project that is focused on Prometheus and
+PromQL seems to be a logical starting point. Most open source and vendored
+solutions in the CNCF Monitoring landscape support both Prometheus ingestion
+format as well as PromQL. 
+
+Perhaps over time, the project can look to add other industry standard
+monitoring protocol support like Graphite or expand into visualization for
+logging and distributed traces, but to start, the focus will be on deriving
+value for one use case (Prometheus/PromQL) and building up from there versus
+attempting to replicate all of the various visualization capabilities of
+Grafana.
+
+The project will also improve the state of the art for finding and choosing
+Prometheus metrics to visualize, and for constructing PromQL queries.
+
+## Draft Goals
+
+Goals are broken down into two sections: 1) Dashboards and 2) Ease of querying. 
+
+**Dashboards**:
+Core visualization goals which are focused on visualization of metric data
+based on PromQL and allowing users to create savable and shareable dashboards
+from them:
+
+* Visualization:
+  * Visualization of multiple metric time series data in a single chart from one
+    or more queries.
+  * Ability to compose multiple charts together into a dashboard.
+  * Ability to store, retrieve, update, delete and share dashboards.
+  * Ability to parameterize charts.
+  * Ability to adjust the lookback window.
+  * Ability to choose settings once for a dashboard and have those settings apply
+    to multiple charts in the dashboard. 
+  * Configurable ability to gracefully degrade the visualization behavior when
+    query result sets are very large.
+* Configuration & Management:
+  * Explicit externalized data model.
+  * Ability to manage lifecycle and configuration of dashboard definitions using a
+    GitOps model.
+  * Ability to migrate from other/legacy dashboard formats, specifically including
+    Grafana dashboard definitions.
+* Extensibility & Reuse:
+  * Deliver as both a library and an application.
+  * Charts and dashboards can easily be placed inside other applications and other
+    use cases.
+  * Future: Ability to extend visualizations to other chart types.
+  * Future: Ability to extend query language to others.
+  * Future: Ability to extend visualization to other data types like logs and
+    distributed traces.
+ 
+**Ease of Querying**:
+Goal focused on helping beginners and first time users of PromQL to craft their
+queries. If PromQL is going to be the long term industry standard open source
+query language, many more end users will have to become familiar with it over
+time:
+
+* Visual Query builder to help users construct PromQL queries.
+* Query Analyzer to help users identify inefficiencies in their queries.
+* Prompts and libraries/catalog of common or example queries to help users get
+  started.
+
+The target user would be a JavaScript developer.
```

Comment:
I think it's a mistake indeed. In the initial doc I put a comment and looks like we were agreed on that. For me as well the whole IT landscape can be seen as a target user.

Reply:
I just removed the bit about the audience being a Javascript developer. We don't seem to have consensus for this, and I think it might be confusing for people.

Reply:
OK, I've added the target audience back in with very different language based on the email from @monadic. I've also moved it to the top of the Goals section.

Reply:
For me it's still odd to have a specific target user even after reading the email explaining why it should be JS dev. I don't even know what specific skills they have comparing to other developers. Even backend developer like clean, nice and easy to use website. And since PromQL or any query language avalable are part of the skillset of the JS developer, I don't why we want to focus of this tiny part of the IT people.
But probably I didn't get the point . So if it's clear for others, fine for me as well.

Reply:
I don't see this as focusing on a tiny number of people. I think it expands the number of people dramatically. The point is that people with web front end skills is a low barrier to entry. Think of this is as more like the minimum amount of skill required. A lot of people (like the current Grafana users) will have additional skills and will also be able to easily use these tools. By targeting people with web front end skillset, we make it accessible to them **and** to other people with backend or more advanced programming skills.

Reply:
In that case, we should just say we are targeting user with low programming skill which will induce naturally a website easy to use / to undersand :).
I don't think saying we are targeting JS developer is a low entry barrier. Basically JS or Typescript have the same difficulties than other language like Golang for example. You don't manage the memory yes, but you have many things to now to be able to know it well.
And since each time we are reading we are targeting JS developer is providing more questions than answers and can be insulting for this part of the community because we are assuming it's easy to be a JS developer ( which is not at all), we should just say people with low programming skill. It's more generic without judging a part of the IT community :)

Reply:
@Nexucis - This change was split across 2 different commits (removal: https://github.com/coredashio/community/pull/1/commits/d24866a5bc8411863cea7add226b39ea8dc54540 and  a new section: https://github.com/coredashio/community/pull/1/commits/62c12020cecae9b34db5e9f13e4f66983912ab26) and this bit was moved to the top of the section, so it's a bit hard to find the update. The short answer is that we're no longer saying that we're targeting a Javascript Developer. Here's the new language in the README from the [addition commit](https://github.com/coredashio/community/pull/1/commits/62c12020cecae9b34db5e9f13e4f66983912ab26):

> We aim to make CoreDash easy to use by not requiring highly specialized skills
to be able to benefit from these tools and dashboards. Anyone with basic web
front end skills (Javascript, etc.) should be able to create dashboard widgets
and screens and connect them to a variety of data sources without needing to
write complex code or be a sys admin / DevOps expert. We hope that by making
these technologies more inclusive to people with a wide variety of skills, we
can open the field to a larger set of well established users who create and
customize dashboard screens.